packages/contracts/test/registry/dao-registry.ts

```diff
@@ -29,36 +24,41 @@ describe('DAORegistry', function () {
 
   beforeEach(async function () {
     // Managing DAO
-    managingDAO = await deployNewDao();
+    managingDAO = await deployNewDAO(ownerAddress);
+
+    // Target DAO to be used as an example DAO to be registered
+    targetDao = await deployNewDAO(ownerAddress);
 
     // DAO Registry
     const Registry = await ethers.getContractFactory('DAORegistry');
     registry = await Registry.deploy();
     await registry.initialize(managingDAO.address);
 
     // Grant the `REGISTER_DAO_ROLE` permission to `signers[0]`
-    managingDAO.grant(registry.address, ownerAddress, REGISTER_DAO_ROLE);
+    await managingDAO.grant(registry.address, ownerAddress, REGISTER_DAO_ROLE);
```

Comment:
Same as before, do these await the mempool request to complete? or the whole transaction to be mined as well?